### PR TITLE
Fix display of user's own holding in hitlist

### PR DESCRIPTION
**Pull request by olovy**
Was broken by the early return in c4434a0.
We can always find the holdings in `@reverse/itemOf` for each Instance in the search result. 
So remove the code that does an additional query for the holding.

## Checklist:
- [x] I have run the unit tests. `yarn test:unit`
- [x] I have run the linter. `yarn lint`

### Tickets involved
[LXL-4541](https://kbse.atlassian.net/browse/LXL-4541)

### CODE REVIEW

File: vue-client/src/components/inspector/reverse-relations.vue [hunk 1]
```diff
@@ -112,11 +96,6 @@ export default {
           });
       }, timeoutLength);
     },
-    gotoHolding() {
-      const locationParts = this.myHolding.split('/');
-      const fnurgel = locationParts[locationParts.length - 1];
-      this.$router.push({ path: `/${fnurgel}` });
-    },
```
Comment (olovy) [inline vue-client/src/components/inspector/reverse-relations.vue:119]: dead code

**Comment by jesperengstrom** [comment]
In the search results: Looking good.

On the instance itself, kind of the opposite... Holding displaying correctly on dev:
Skärmavbild 2024-08-14 kl  15 48 08

But not on this branch:
Skärmavbild 2024-08-14 kl  15 48 57

`myHolding` returning `null`, do we need to look for it somewhere else? 🤔

**Comment by olovy** [comment]
> On the instance itself,  ...  `myHolding` returning `null`, do we need to look for it somewhere else? 🤔

Good catch!

The data is framed in the search result but not when opening a record in the inspector.
So then we have to look in quoted (embellished) instead.

**Review by jesperengstrom — APPROVED** [review]
LGTM!

**Comment by olovy** [comment]
A further improvement would be to display the holding link directly without the spinner when viewing the record.

**Pull request merged**